common/src/test/java/org/conscrypt/javax/net/ssl/SSLContextTest.java

```diff
@@ -664,14 +665,21 @@ public void test_SSLContext_SSLv3Unsupported() throws Exception {
     }
 
     private static void assertContentsInOrder(List<String> expected, String... actual) {
+        List<String> actualList = Arrays.asList(actual);
         if (expected.size() != actual.length) {
             fail("Unexpected length. Expected len <" + expected.size() + ">, actual len <"
                     + actual.length + ">, expected <" + expected + ">, actual <"
-                    + Arrays.asList(actual) + ">");
+                    + actualList + ">");
         }
-        if (!expected.equals(Arrays.asList(actual))) {
-            fail("Unexpected element(s). Expected <" + expected + ">, actual <"
-                    + Arrays.asList(actual) + ">");
+
+        if (System.getProperty("os.name", "").toLowerCase().startsWith("windows")) {
+            // CpuFeatures.isAESHardwareAccelerated is not reliable on windows
```

Comment:
nit: I don't think that's the root cause, otherwise `test_SSLContext_pskOnlyConfiguration_defaultProviderOnly` should fail too.

Also, please add a TODO and ideally a bug (internal, assigned to me) to review this in a few months.  It only started happening recently so may well be a bug in the Windows provider which will get fixed.

Reply:
The PSK ciphers are not conditional on the AES support as defined, so I think it's related.  Should they be?